docs/quickstart.rst

```diff
@@ -0,0 +1,385 @@
+        to finish processing, then downloads it to the path specified.
+
+        Returns:
+            Dataset
+
+        Parameters:
+            path (str): the path that the Dataset should be downloaded to
+
+            email_address (str): the email address that will be contacted with info
+                                    related to the dataset
+            
+            dataset_dict (dict): a fully formed Dataset `data` attribute in the form:
+                                    {
+                                        "Experiment": [
+                                            "Sample",
+                                            "Sample"
+                                        ]
+                                    }
+                                    use this parameter if you want to specify specific Samples
+                                    for your dataset
+                                    each part of the dict can be a pyrefinebio object or an accession
+                                    code as a string
+
+            experiments (list): a list of Experiments that should be added to the dataset
+                                use this parameter if you only care about the Experiments - all 
+                                available samples related to each Experiment will be added  
+                                the list can contain Experiment objects or accession codes as strings
+
+            aggregation (str): how the Dataset should be aggregated - by `EXPERIMENT` or by `SPECIES`
+
+            transformation (str): the transformation for the dataset - `NONE`, `MINMAX`, or `STANDARD`
+
+            skip_quantile_normalization (bool): control whether or not the dataset should skip quantile
+                                                normalization for RNA-seq Samples
+
+            extract (bool): if true, the downloaded zip file will be automatically extracted
+
+            prompt (bool): if true, will prompt before downloading files bigger than 1GB
+
+You can also get info on class methods by passing in `class.method` to the help function.
+
+Here's an example:
+
+.. code-block:: shell
+
+    >>> import pyrefinebio
+    >>> pyrefinebio.help("Sample.search")
+
+Using the CLI
+-------------
+
+pyrefinebio also provides a CLI that exposes the `download_dataset()`, `download_compendium()`, `download_quandfile_compendium()`, and `help()` functions.
+
+Each function has its own command: `download-dataset`, `download-compendium`, `download-quandfile-compendium`, and `describe`, respectively.
+
+To use the CLI just type `refinebio COMMAND` into a shell.
+
+You can pass in the option `--help` to each command to get more information about it as well.
+
+Here's an example of downloading a Dataset using the CLI:
+
+.. code-block:: shell
+
+    $ refinebio download-dataset --path "./path/to/dataset.zip" --email-address "foo@bar.com" --dataset-dict '{"GSE74410": ["ALL"]}'
+
```

Comment:
I think we should have a CLI way under each of the function headings above instead of a separate brief section.

I think CLI would be preferable than python in the absence of an R package and it is worth providing fleshed out examples for it.

Reply:
I added a whole page for CLI. lmk what you think!